src/pages/deployment/environment-variables.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Environment Variables
+---
+
+Railway provides the following additional system environment variables to all 
+builds and deployments.
+
+| Name                         | Description |
+|------------------------------|-------------|
+| `RAILWAY_DEPLOYMENT_DOMAIN`  | The domain specific to the current deployment, of the form `c56c8f2a83d2.up.railway.app` (not available during builds) |
```

Comment:
So, if we switch to deploymentID this could be available during builds

Reply:
But I suppose we can update this later when it's actually avail

Reply:
Turns out it wasn't even available to the deploy because it's generated after. Removing for now until after the domains refactor.